index.html

```diff
@@ -0,0 +1,121 @@
+          <li><i class="fa fa-brands fa-twitter"></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <div class="navbar">
+        <a class="my-logo" href="#headline">YouCode</a>
+        <ul class="menu">
+          <li><a class="menuItem" href="about.html">About</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Program</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Join</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Sponsor</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">News</a></li>
+        </ul>
+        <ul class="desktop-menu">
+          <li><a class="" href="about.html">About</a></li>
+          <li><a class="" href="#"style="color: lightgray">Program</a></li>
+          <li><a class="" href="#" style="color: lightgray">Join</a></li>
+          <li><a class="" href="#" style="color: lightgray">Sponsor</a></li>
+          <li><a class="" href="#" style="color: lightgray">News</a></li>
+          <button class="navbar-btn">CC Campaign</button>
+         </ul>
+        <button class="hamburger">
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+      </div>
+    </header>
+    <main id="headline" class="headline">
+      <p class="greet-title">"Hello! Sharing world!"</p>
+      <h2 class="title">A WEB DEVELOPMENT COURSE TO GET YOU STARTED</h2>
+      <p class="content">
+        Kickstart your courses in web development. Master html, css and javascript
+        courses using hands-on projects involving HTML, JavaScript, React.js, Ruby and more. 
+        No prior experience required.
+      </p>
+      <h2 class="date-set">2023.02.01(MON) - 15(THUR)</h2>
+      <p class="location">(Addis Ababa Arat Kilo Campus IT hall)</p>
+    </main>
+    <section class="main-program-list">
+      <h2 class="main-program-title">Main Program</h2>
+      <hr>
+      <section class="programs">
+          <section class="program-one">
+            <img class="program-tag" src="images/html.png" alt="Html logo">
+            <h2 class="program-title">HTML</h2>
+            <p class="program-content">Learn how to use CSS to style and visually orginize HTML pages</p>
+          </section>
+          <section class="program-two">
+            <img class="program-tag" src="images/css.png" alt="css logo">
+            <h2 class="program-title">CSS</h2>
+            <p class="program-content">Learn how to use CSS to style and visually orginize HTML pages</p>
+          </section>
+          <section class="program-three">
+            <img class="program-tag" src="images/js.png" alt="javascript logo">
+            <h2 class="program-title">JS</h2>
+            <p class="program-content">Learn how to use CSS to style and visually orginize HTML pages</p>
+          </section>
```

Comment:
- [ ] I think it would be better if you improve the design of the above section to match the one from the template such as `height` of each card, `height of the whole section`, `number of cards`, `fonts(size, weight,...) on some text`, etc.
### Template Design 
![Screenshot from 2023-02-14 01-29-57](https://user-images.githubusercontent.com/83042742/218598860-922b7734-e620-43a0-a76c-8594eb85cfe4.png)

### Your Design 
![Screenshot from 2023-02-14 01-34-19](https://user-images.githubusercontent.com/83042742/218598899-5c6cd583-676c-4eaf-9934-fec3512c7bd6.png)